From 31125302111565bc32d938cda7d95d31d7643317 Mon Sep 17 00:00:00 2001
From: Graeme Smecher <gsmecher@t0.technology>
Date: Tue, 22 Jul 2025 13:48:14 -0700
Subject: [PATCH] Workaround for pre-PEP 621 setuptools

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 57546a5..cab8e56 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -20,7 +20,7 @@ classifiers=[
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
-license = "BSD-3-Clause"
+license = {file="LICENSE"}
 dependencies = ["requests", "requests-futures"]
 dynamic = ["version", "scripts"]
 
-- 
2.47.2

